rasa/core/events/__init__.py

```diff
@@ -1167,6 +1166,23 @@ def apply_to(self, tracker: "DialogueStateTracker") -> None:
         tracker.change_form_to(self.name)
 
 
+class Form(ActiveLoop):
+    """Legacy handler of old `Form` events.
+
+    The `ActiveLoop` event used to be called `Form`. This class is there to handle old
+    legacy events which were stored with the old type name `form`.
+    """
+
+    type_name = "form"
+
+    def as_dict(self) -> Dict[Text, Any]:
+        d = super().as_dict()
+        # Dump old `Form` events as `ActiveLoop` events instead of keeping the old
+        # event type.
+        d["event"] = ActiveLoop.type_name
+        return d
+
+
 class FormValidation(Event):
     """Event added by FormPolicy to notify form action
```

Comment:
we should remove reference to the formpolicy here for sure

Reply:
Kept it and added `RulePolicy` as this is still true

Reply:
wait but i thought that `FormPolicy` is dead?

Reply:
It's deprecated, not dead.